src/lib/kit/components/Card/Card.tsx

```diff
@@ -112,7 +120,9 @@ export const Card: React.FC<CardProps> = ({
                 className={b('header', {interactive: Boolean(handleHeaderToggle)})}
                 onClick={handleHeaderToggle}
             >
-                <div className={b('header-left')}>{title}</div>
+                <div ref={titleRef} className={b('header-left')}>
```

Comment:
ref should be on another div 
right here
`<div className={b('title')}>{propsTitle}</div>`
otherwise it will be like this
<img width="1680" alt="image" src="https://github.com/gravity-ui/dynamic-forms/assets/23384737/73ed8d0c-85f0-441c-a9d8-a6284f0ec4d3">